fwd-template.json

```diff
@@ -24,6 +24,30 @@
                     "path": "template/Dockerfile"
                 }
             ]
+        },
+        {
+            "name": "8.0-nginx-oci8-prod",
```

Comment:
you need to add here without prod too... for both versions, and run template again

Reply:
Check now please.